modules/navier_stokes/src/materials/GeneralFunctorFluidProps.C

```diff
@@ -186,21 +186,21 @@ GeneralFunctorFluidProps::GeneralFunctorFluidProps(const InputParameters & param
   _Pr.setFunctor(_mesh, blockIDs(), [this](const auto & r, const auto & t) -> ADReal {
     static constexpr Real small_number = 1e-8;
 
-    return fp::prandtl(_cp(r, t), _mu(r, t), std::max(_k(r, t), small_number));
+    return fp::prandtl(_cp(r, t), _mu_rampdown(r, t) * _mu(r, t), std::max(_k(r, t), small_number));
```

Comment:
I don't think this is correct. You're calling `_mu` here which will already have the application of `_mu_rampdown`

Reply:
ah good catch. I ll put it in the next PR. 
we need a real customer for all these properties inside or outside the framework, this would have been caught immediately if fed into a correlation